p2p/protocol/handler.go

```diff
@@ -46,31 +45,25 @@ func QuaiProtocolHandler(stream network.Stream, node QuaiP2PNode) {
 	// Create a channel for messages
 	msgChan := make(chan []byte, msgChanSize)
 	full := 0
-
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	var once sync.Once
 	// Start worker goroutines
```

Comment:
Please update to single goroutine. This is because each protocolhandler only handles messages from one peer correct?